tests/unit/callbacks/test_logging.py

```diff
@@ -173,39 +173,22 @@ def test_extract_from_logs():
     }
 
     value = extract_from_logs('value', logs)
-    if value != 0:
-        raise ValueError('We should find values in logs')
+    assert value == 0, 'We should find values in logs'
 
     nothing = extract_from_logs('nothing', logs)
-    if nothing != 'N/A':
-        raise ValueError('Nothing should return N/A')
+    assert nothing == 'N/A', 'Nothing should return N/A'
 
     specific_value = extract_from_logs('specific_value', logs)
-    if specific_value != 42:
-        raise ValueError(
-            'If no agent is specified, it should find any specific_value in agent.'
-            f'Here specific_value was {specific_value} instead of 42.'
-        )
+    assert specific_value == 42, 'If no agent is specified, it should find any specific_value in agent. Here specific_value was {specific_value} instead of 42.'
```

Comment:
Line too long no ?

Reply:
Yes, but shouldn't it been raised by the linter ?